packages/parser/test/Parser_test.re

```diff
@@ -34,28 +34,29 @@ let error_tests_data =
     ),
     (
       "@media $",
-      "Parse error while reading token '$' on line 1 at position 6",
-    ),
-    (
-      /* whitespace must follow `not` */
-      "@media (not(color)){}",
-      "Parse error while reading token ')' on line 1 at position 17",
-    ),
-    (
-      /* whitespace must follow `or` */
-      "@media (not (color)) or(hover) {}",
-      "Parse error while reading token 'or(' on line 1 at position 20",
-    ),
-    (
-      /* whitespace must follow `and` */
-      "@media (not (color)) and(hover) {}",
-      "Parse error while reading token 'and(' on line 1 at position 20",
-    ),
-    (
-      /* space between < and = is invalid */
-      "@media (width < = 33px) {}",
-      "Parse error while reading token '=' on line 1 at position 15",
+      "Parse error while reading token '' on line 1 at position 7",
```

Comment:
weird error message; it should at least render the `$`.

Reply:
huh yeah, dunno why this is the case